src/build.py

```diff
@@ -751,7 +751,10 @@ def GetRepoInfo():
 
 def OverrideCMakeCompiler():
   if IsWindows():
-    return []
+    MSVC_COMPILER = os.path.join(os.environ['GYP_MSVS_OVERRIDE_PATH'],
```

Comment:
It is guaranteed to exist after running `vs_toolchain.py` (which always run before building anything).